__tests__/CreateAgencyBtn.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render, fireEvent } from '@testing-library/react';
+import CreateAgencyBtn from "../src/components/CreateAgencyBtn/CreateAgencyBtn";
+
+const mockHistoryPush = jest.fn();
+
+// Creates a mock router to be used for testing
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useHistory: () => ({
+        push: mockHistoryPush,
+    }),
+}));
+
+// Checks if the rendered button from the CreateAgencyBtn component redirects to the correct URL
+describe('CreateAgencyBtn', () => {
+    it('Button goes to the correct URL on click', () => {
+      const { getByRole } = render(
```

Comment:
It looks like your Javascript has 4-space indents but your JSX has 2-space indents. I think we should make them match—Emmanuel and I used 2 spaces for all the form components

Reply:
Agreed, let's keep a consistency in indentation with 2 spaces = 1 tab

Reply:
Yea, sorry about that, styling slipped our minds for some of the parts. I'll implement the changes sometime today and make sure everything is consistent.